es-bs-base/scss/_variables.scss

```diff
@@ -1037,7 +1037,7 @@ $badge-pill-border-radius:          10rem !default;
 // Modals
 
 // Padding applied to the modal body
-$modal-inner-padding:               2rem !default;
+$modal-inner-padding:               0 2rem 2rem !default;
```

Comment:
nit: could you write this as `<top> <right> <bottom> <left>`? I have trouble remembering if the short-hand for 3 variables means `<top-bottom> <left> <right>` or `<top> <bottom> <left-right>`